pyro/distributions/constraints.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2017-2019 Uber Technologies, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
+import torch
 from torch.distributions.constraints import *  # noqa F403
 from torch.distributions.constraints import Constraint
 from torch.distributions.constraints import __all__ as torch_constraints
```

Comment:
Kind of off-topic, but I noticed there are a couple `Constraint` implementations here marked with `# TODO move this upstream to torch.distributions` that are several months old... perhaps that was forgotten about somewhere along the way? :laughing:

Reply:
Haha, if I remember correctly, then the blocker is it is not straightforward to add write tests when we move those upstream.

Reply:
Yes, I have halted all my "move upstream" work #2630 until we split up test_distributions.py and can use PyTest https://github.com/pytorch/pytorch/issues/11578